pkg/util/runtime/cpu_notlinux.go

```diff
@@ -26,4 +26,4 @@ import (
 // NumCPU ...
 func NumCPU() int {
 	return runtime.NumCPU()
-}
+}
```

Comment:
Can you fix the typo to pass the "gofmt -s" check?

Reply:
Fixed